### PR TITLE
RESTWS-794:Unused assignments should be removed

### PR DESCRIPTION
**Pull request by gracebish**
**Pull request title**
RESTWS-794:Unused assignments should be removed
**JiraIssueNumber:JiraIssueTitle**
RESTWS-794:Unused assignments should be removed 
**Description of what I changed**
Remove an unused assignment
**Issue I worked on**
https://issues.openmrs.org/browse/RESTWS-794
## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

  No? -> [read here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to squash multiple commits into one

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [x] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

File: omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/PatientIdentifierResource1_8.java [hunk 1]
```diff
@@ -110,7 +110,6 @@ public void setIdentifierType(PatientIdentifier instance, PatientIdentifierType
 			PatientIdentifierType pit = Context.getPatientService().getPatientIdentifierTypeByName(name);
 			if (pit != null) {
 				instance.setIdentifierType(pit);
-				reset = true;
```
Comment (HerbertYiga) [inline omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/PatientIdentifierResource1_8.java:113]: is this flag not used in any place in the entire class?

Comment (gracebish) [reply omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/PatientIdentifierResource1_8.java:113]: Could you kindly point where?

Comment (gracebish) [reply omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/PatientIdentifierResource1_8.java:113]: @sacull,Kindly review

Comment (sacull) [reply omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/PatientIdentifierResource1_8.java:113]: > is this flag not used in any place in the entire class?

This is a local variable, so its scope is only a method, not an entire class. In addition, it is changed at the very end of this method, so the method ends after the change and access to this variable is lost. So why waste CPU cycles to change this variable?

**Comment by sacull** [comment]
@gracebish Where did you add the tests, because I look and can't see. ;)
From the description of PR:
image

**Review by sacull — APPROVED** [review]
It looks good to me.

**Comment by sacull** [comment]
@gracebish By the way, remember that intro tickets are mostly for newcomers to familiarize themselves with Jira, Github, and the OpenMRS workflow. You don't need it anymore. ;)

**Comment by gracebish** [comment]
@sacull, Have you looked at
https://wiki.openmrs.org/display/RES/OpenMRS+Developer+Stages vs my
portifolio?

**Comment by sacull** [comment]
Maybe your achievements are modest, but Github and Jira are no longer a challenge for you. Let the OpenMRS code begin to be that challenge. ;)

**Pull request merged**